.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+**Description**
+A description of the PR, should include a decent explanation as to why this change was needed and a decent explanation as to what this change does
+
+**Review Instructions**
+Describe if this ticket needs review and if so, how one may go about it in qa and/or staging environments.
+For example, a ticket based on Security Hub, Snyk, or Dependabot may not need review since those services 
+will generate new warnings if the issue has not been resolved properly. On the other hand, an infrastructure
+ticket that results in visible changes to the end-user will definitely require review. 
+Many tickets will likely be between these two extremes, so some judgement may be required.
+
+**Issue**
+A link to a github issue or SEAB- ticket (using that as a prefix)
+
+**Security**
+If there are any concerns that require extra attention from the security team, highlight them here.
+
+Please make sure that you've checked the following before submitting your pull request. Thanks!
+
+- [ ] Check that you pass the basic style checks and unit tests by running `mvn clean install`
```

Comment:
Nit (although it'll probably be helpful to new coop students)

For this repo I know there are several different maven projects. For example,

- https://github.com/dockstore/dockstore-support/tree/develop/tooltester
- https://github.com/dockstore/dockstore-support/tree/develop/metricsaggregator

And for this repo you can't run `mvn clean install` in the base of the repo. So perhaps change this so it asks the developer to run `mvn clean install` in the directory that they modified.

Reply:
Fair, I'll add a reference to https://ucsc-cgl.atlassian.net/browse/SEAB-5300 as well since it probably  shouldn't have multiple projects this way